pinot-core/src/main/java/com/linkedin/pinot/core/startree/OffHeapStarTreeBuilder.java

```diff
@@ -139,7 +139,7 @@ public void init(StarTreeBuilderConfig builderConfig) throws Exception {
     if (outDir == null) {
       outDir = new File(System.getProperty("java.io.tmpdir"), V1Constants.STAR_TREE_INDEX_DIR + "_" + DateTime.now());
     }
-    LOG.debug("Index output directory:{}", outDir);
+    LOG.info("Index output directory:{}", outDir);
```

Comment:
This is made intentionally. I believe it should be at "info" level since it does not contain any of the debug message, e.g. intermediate results. And similar messages in other classes are also at info level.